test-harness/src/test/java/com/netflix/conductor/tests/integration/AbstractEndToEndTest.java

```diff
@@ -0,0 +1,65 @@
+package com.netflix.conductor.tests.integration;
+
+import com.netflix.conductor.common.metadata.tasks.TaskDef;
+import com.netflix.conductor.common.metadata.workflow.TaskType;
+import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
+import com.netflix.conductor.common.metadata.workflow.WorkflowTask;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+public abstract class AbstractEndToEndTest {
+
+    private static final String TASK_DEFINITION_PREFIX = "task_";
+    private static final String DEFAULT_DESCRIPTION = "description";
+    // Represents null value deserialized from the redis in memory db
+    private static final String DEFAULT_NULL_VALUE = "null";
+
+    protected WorkflowTask createWorkflowTask(String name) {
+        WorkflowTask workflowTask = new WorkflowTask();
+        workflowTask.setName(name);
+        workflowTask.setWorkflowTaskType(TaskType.SIMPLE);
+        workflowTask.setTaskReferenceName(name);
+        workflowTask.setDescription(getDefaultDescription(name));
+        workflowTask.setDynamicTaskNameParam(DEFAULT_NULL_VALUE);
```

Comment:
This default null values population are necessary as the readJson mechanism populates this attributes. In order to use `WorkflowDef.equals` on the tests these need to be populated.